.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+__*This is the standard PR description template.*__
+
+__*Delete the parts of this template that aren’t relevant to the current PR you are submitting – for example, you may not need to amend documentation or testing of the component.*__
+
+---
+
+_A short description of your PR goes here._
+
+## UI Review Checks
+
+![image](https://media.github.je-labs.com/user/326/files/c1e6632c-e23e-11e6-91e2-4625e4f89d6d)
+
+
+- [ ] This PR has been checked with regard to our brand guidelines
+- [ ] UI Documentation has been [created|updated]
+- [ ] This code has been checked with regard to our accessibility standards
+  - [ ] HTML is valid [[link]](https://docs.google.com/document/d/17Sr6m08eYrTKX1Q9HCoHj6nAeQ3Tx2Ix589_kebSDnY/edit#heading=h.8hhpeziqc7vf)
```

Comment:
Is this doc publicly readable?

Reply:
No it isn't. The a11y info lives in two (three) places at the moment, none of which are accessible to the public. We should really decide where this content should go and link to that? Maybe the global component library?

Reply:
Yep – it should definitely live in the component library docs 👍

Reply:
I've added this doc into a [PR for the fozzie docs](https://github.com/justeat/global-component-library/pull/30) to make this publically available